aws-aurora-postgresql.yml

```diff
@@ -52,12 +52,10 @@ provision:
   - <<: *nullable_number
     field_name: serverless_max_capacity
     details: The maximum capacity for the cluster. Must be greater than or equal to `serverless_min_capacity`. Valid capacity values are in a range of 0.5 up to 128 in steps of 0.5.
-  - <<: &nullable_string
-      type: string
-      default: null
-      nullable: true
-    field_name: engine_version
-    details: The Aurora engine version, e.g. "13.7". Not all features are supported by all versions. Refer to the AWS documentation for more details.
+  - field_name: engine_version
+    type: string
+    default: "14.4"
```

Comment:
**question**: Why have we decided to set a default version? Are we planning to maintain that default? Additionally, it doesn't seem to match the default in the console.

Reply:
Good question. I totally agree with you. Since we need the engine version in order to create the correct parameter group, I will make the property part of the plan inputs and remove the default version. What do you think @jhvhs ?